sw/cmake/toolchain-llvm_origin.cmake

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 ETH Zurich and University of Bologna.
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
```

Comment:
Don't do this in general; your contribution should be as clean as possible and not affect any existing setups